<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtuLineLossDialog</class>
 <widget class="QDialog" name="dtuLineLossDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>停用</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>513</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lab1">
     <property name="text">
      <string>通信模式：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="comMode">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>非平衡模式</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>平衡模式</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>438</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>链路地址：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="spinADDR">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>475</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="SerialDialog" name="widgetSerial" native="true"/>
   </item>
   <item row="3" column="4">
    <widget class="ProtoDialog" name="widgetProto" native="true"/>
   </item>
   <item row="4" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>408</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>321</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>321</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialDialog</class>
   <extends>QWidget</extends>
   <header location="global">serialdialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProtoDialog</class>
   <extends>QWidget</extends>
   <header location="global">protodialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnRead</sender>
   <signal>clicked()</signal>
   <receiver>dtuLineLossDialog</receiver>
   <slot>read()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>dtuLineLossDialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>read()</slot>
  <slot>save()</slot>
 </slots>
</ui>
